bsip-0083.md

```diff
@@ -0,0 +1,80 @@
+
+Therefore, this BSIP defines a process for BitAssets management to ensure protocol security remains the singular focus of the block producer role, and give BitAssets the best chance for long-term viability by nominating a new custodian to quickly react to market conditions.
+
+# Motivation
+Block production is an essential component of the BitShares platform itself, and as such should be expected to take an agnostic and neutral view towards the products deployed within the platform. Therefore, this BSIP will define a process for BitAsset management to ensure protocol security remains the singular focus of the block producer role and give BitAssets the best chance for long-term stability by nominating a new custodian to quickly react to market conditions. In order for the distributed consensus mechanism of DPOS to achieve adequate decentralization, it is imperative that block producers maintain strict independence from the politics of individual products deployed on the platform. 
+
+In short, this BSIP will put the BitAssets under the control of a new user account controlled by the stakeholders through a new type of voting. With this change, the newly appointed BitAssets manager will be able to independently maintain the parameters of their derivative asset without interference from the platform’s block producers and committee members. At the same time, the platform itself and its committee will not be unduly influenced by the demands of private product management. An advantage gained by this is that the BitAsset manager will be able to react appropriately to rapidly changing market conditions, as may be required to properly maintain a successful derivative asset.
+
+# Rationale
+Because the BitAssets are currently controlled by platform-wide governance mechanisms, and because these are not well suited to the management of smart coins, which may at times need to respond quickly to market conditions, the authors propose to reorganize these assets to utilize a dedicated stakeholder-voted multisignature authority and thereby enable these assets to update in synchronization with market ebbs and flows.
+
+The block producers will no longer provide feeds for BitAssets.
+
+# Specification
+The following technical changes are required:
+
+**Assignment of Accumulated Fees**
+
+At the hardfork date, the accumulated fees of the existing BitAssets will be transferred to the committee-account.
+
+**Creation of BITASSET.MANAGER asset**
+
+At the hardfork date, a new BITASSET.MANAGER asset will be created with the committee account as its owner and all flags and permissions disabled. The initial distribution will be made equal to the current BTS distribution. BTS that are locked up in open orders, in collateral positions, in HTLC objects or in vesting balances will be assigned to the owners of the respective positions. Other BTS positions are out of scope (e. g. stealth).
+Subsequently, tokens of this new asset can be freely transferred or traded.
```

Comment:
Also air drop to centralized exchanges, BEOS custodian accounts and etc? Great efforts on communications may be needed to make this protocol upgrade going smoothly.

Reply:
Yes. It is up to the users who hold their BTS on CEXs or with BEOS to care about their voting rights. Most probably don't *want* to vote anyway, because otherwise they wouldn't do this.

Also, this will not be the first time that users will be pre-warned to hold funds in their own account if they want to participate in a sharedrop.

Reply:
I don't think the “Creation of BITASSET.MANAGER asset” is necessary.

This will also push the bitasset to be a third party, it's no different with before words.

What's the right method is which the stakeholder of BTS vote the price feeder and committee of bitasset.